### PR TITLE
Add namespace support

### PR DESCRIPTION
**Pull request by corest**
Following https://github.com/giantswarm/app/pull/9 and https://github.com/giantswarm/architect/pull/398

### CODE REVIEW

**Comment by corest** [comment]
Real-life example https://github.com/giantswarm/g8s-cert-manager/pull/13

**Comment by corest** [comment]
Btw, how can I test it without merging?

**Comment by tfussell** [comment]
> Btw, how can I test it without merging?

You can use a branch of the architect-orb in another repo like this https://github.com/giantswarm/releases/pull/158/files

File: src/jobs/push-to-app-collection.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ parameters:
   app_name:
     description: "Name of the application."
     type: "string"
+  app_namespace:
+    description: "Namespace of the application."
```
Comment (kopiczko) [inline src/jobs/push-to-app-collection.yaml:12]:
```suggestion
    default: "giantswarm"
    description: "Namespace of the application."
```

**Comment by corest** [comment]
Tested changes, works as expected

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.7] - 2020-04-08
+
+### Added
+
+- New parameter `namespace` for `push-to-app-collection` command, which allows
```
Comment (kopiczko) [inline CHANGELOG.md:16]:
```suggestion
- Add new parameter `namespace` to `push-to-app-collection` job, which allows
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.7] - 2020-04-08
```
Comment (kopiczko) [inline CHANGELOG.md:12]: I'd ask to create a release in a separate PR. Other than that LGTM.

**Pull request merged**